apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/GraphQLCompiler.kt

```diff
@@ -84,6 +85,7 @@ class GraphQLCompiler {
       val nullableValueType: NullableValueType,
       val useSemanticNaming: Boolean,
       val generateModelBuilder: Boolean,
+      val prependGetter: Boolean,
```

Comment:
should we call this: `useJavaBeansSemanticNaming`?

Reply:
👍